_rules/form-field-non-empty-accessible-name-e086e5.md

```diff
@@ -45,10 +45,10 @@ _There are currently no assumptions_
 
 The list of roles in the applicability is derived by taking all the roles from [WAI-ARIA Specifications](#wai-aria-specifications) that:
 
-- have [semantic roles][] that inherit from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and
+- have [semantic roles][] that inherit from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input`, `select` or `menuitem` role, and
```

Comment:
`menuitem` isn't an abstract role. I don't think this needs to be said explicitly.

```suggestion
- have [semantic roles][] that inherit from the `input`, `select` or `menuitem` role, and
```